django/db/backends/base/schema.py

```diff
@@ -774,15 +774,15 @@ def remove_field(self, model, field):
             ):
                 self.deferred_sql.remove(sql)
 
-    def alter_field(self, model, old_field, new_field, strict=False):
+    def alter_field(self, model, old_field, new_field, strict=False, force=False):
         """
         Allow a field's type, uniqueness, nullability, default, column,
         constraints, etc. to be modified.
         `old_field` is required to compute the necessary changes.
         If `strict` is True, raise errors if the old column does not match
         `old_field` precisely.
         """
-        if not self._field_should_be_altered(old_field, new_field):
+        if not self._field_should_be_altered(old_field, new_field) and force is False:
```

Comment:
Cheaper to evaluate a bool than call a method that performs field deconstruction.

```suggestion
        if not (force or self._field_should_be_altered(old_field, new_field))
```